doc/man3/RSA_public_encrypt.pod

```diff
@@ -46,17 +48,21 @@ Encrypting user data directly with RSA is insecure.
 
 =back
 
-B<flen> must be less than RSA_size(B<rsa>) - 11 for the PKCS #1 v1.5
-based padding modes, less than RSA_size(B<rsa>) - 41 for
+B<flen> must not be more than RSA_size(B<rsa>) - 11 for the PKCS #1 v1.5
+based padding modes, not more than RSA_size(B<rsa>) - 42 for
 RSA_PKCS1_OAEP_PADDING and exactly RSA_size(B<rsa>) for RSA_NO_PADDING.
 The random number generator must be seeded prior to calling
 RSA_public_encrypt().
```

Comment:
I think this comment about the RNG can be removed.

Reply:
But only for 1.1.1+ I guess

Reply:
It does not really hurt to know that this function can fail if the random number generator fails.
Maybe it would be good to say, that one cannot expect the same result when the
encrypt function is called a second time?

Reply:
BTW: also RSA_private_decrypt also uses random data, for the blinding, but the result does not look
different, though.

Reply:
I wonder if the blinding is does not have a much larger variation on the micro-timing than the
constant time code which was added in the PKCS1 and OAEP code path.

Reply:
and the padding bytes, need to call the random generator again, if a 0 was returned.
That are all random timing differences, which should make an attackers life miserable.